src/core/config.go

```diff
@@ -1027,6 +1025,13 @@ func (config *Configuration) Completions(prefix string) []flags.Completion {
 	return ret
 }
 
+var aliasInvocationCount = metrics.NewCounterVec(
+	"alias",
+	"invocation",
+	"How many times each alias is used",
+	[]string{"alias"},
+)
```

Comment:
Ordinarily I would recommend that something initialises all valid metric labels here to zero, otherwise the first hit isn't counted. But because this is pushing to an aggregation gateway, I suppose that is not required!

Reply:
indeed